util/password/password.go

```diff
@@ -47,16 +46,11 @@ func GetConfirmedPassword() ([]byte, error) {
 func GetAccountPassword() ([]byte, error) {
 	var passwd []byte
 	var err error
-	var pstr string
-	flag.StringVar(&pstr, "p", "", "wallet password")
-	flag.Parse()
-	if pstr == "" {
-		passwd, err = GetPassword()
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		passwd = []byte(pstr)
+
+	// Don't support input passwd in os.Args as plain text
```

Comment:
The CLI has a -p params now, which seems to be conflicting with this comment

Reply:
cmdline  -p params is NOT safe. The password would be let out via bash_history.
I suggest cancel the -p flag, only input password via GetPassword() which without echo.

Reply:
Agree. Or we can keep it but only use it for testnet cluster (not put it in help)